### DISCUSSION TITLE
Sidebar has an extra shadow on mobile

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#6339] [Help]
I don't know if this is a problem or how it should be, but the sidebar has an extra shadow on mobile. I wanted to submit a PR but couldn't build the assets.

![7](https://user-images.githubusercontent.com/29732308/234939624-b1384ff1-9eda-4c27-83d7-179d0b41a1b3.png)

This is the solution.
![71](https://user-images.githubusercontent.com/29732308/234939917-3ec3bb0c-9fdb-4352-b9a4-67783d679c31.png)

**Comment by danharrin** [comment]
The assets can be built using `npm run build`

**Reply by ArtMin96** [reply]
Ok. I will try
